tutorials/ndvi_hotspots.ipynb

```diff
@@ -0,0 +1,1063 @@
bmqGrND1EkxCLArtZkFeZuSqYO0K8x/nv5eqMOw7aGdpRlVvKXeK7Cb847+9f/yrhiKyVgx9h2oF7sbBNH9p42LuCI4PFyY/0HcRwsSgFTlngo9Iq1hv1viPA9NhIAwR2Sia1yvIgaH3pI8e4RRZg4iFzg1zZGohMHaFeVVTfEJMGX+JpMkjLgK5dZQ+II8jQlpSzGQCdav46f/+A0po7MLS74+4lxXRJ2SlkZWg+ECKE/ZqonKOPEbWv22ITx3+NJL30xzfvD8RP5xxu5qCIB0H0iUQpo6JiLpZI29qSpjp87W8Jrzr6A+e6Txity1Ns0FrzfDTCeHmzyw+dFzu9+AE+hcY2Ph+ompq5KKCSpBPEyUOMwU4Cbr7A/1fLoSFIr0euXy44P0Z5SFPE+u/e45b1lSblvN5IulAvd6SDpHtyzVqtSBWGeKEFmKeoHce/xRwV7NFQQaBbApqKRj+8kQYCuH+SHwYUO9GzFLiPw3sfzggtGDZtoggqUc3bypMRb2ooVWziUBkbr6/RfagpEYuHf3TSPfP7wm9RzYCs3OIM4Sux9QCvWxhWchioBAZP5wwu5rp5xEtLSbU6KomXAJln+nGM+GcMVeaqq0pgnkYtZCELlFt7QxUE4XzwxGhMtOQiEowjJ5qUaOGgjBlnoQHOH15RF/PQN0kR5R19Mee835PngAlqXdrhssJXw+YlWZZ/foNpLSQFrMNY/ntC6Q2lJIotkFZhakdiZGwD6SUOLQdZelRpUYsKkRUhE+ecA709z3N9YbrV7cIp3HWMd2f4TJvnKIfEVbCBHphqVdrtosK2zq0hDwVii6Ii5pBwcqQh0yJGX+MuGAwy4b/y//5/4YiY9eScBmovzLkMVPfbma98kMmGo9sDGJlGA89t9/d4r0nJT8D1IJkCJESNXplUTJTKjmrLg1kU5BXCtqMiIXJgouW7oeeIgyX0wl37WjdGq1b+p8O5MEjBfjHkfHpCEWSU2bqR4oXNLdrdK3JRPLk4VIoRqMXhh8//xNDOhNPiYu4sP/4RBQekSPhNNH+bo0UiqZd4i8XZCuoFwvGy0huBbqd4bJNv0SoTL2umc4DvpuQzRwJFI2ECugE/umCKBI6gU0VciiopcE/9EyPB6TUrKsF2SRcml+yLGC5W5HbSLtqYQyory1MBiMc8dNEaWb+RTpP6CKpby3xIeGHhMrzcCTmhEwCYiIcJrSyTI8DRQuqqkbvFmQirqw4x4509uQUWC5WiJjRyhCdonQau1pw330hHTNaOrSLxPNEo2vC1BOIpAA+Zaq1m9MjQ49rNeREIyuEzRwev7D/fMd09tg2Y9aOoDzDOQCSm9tXv2qdKeln+8+LFavfX2NUNdtNtEAwK7mncCY8eBSGMydyJenvPKrREBXTD2f8vqd/irQvam6+34HOyFqRP0akl+hSEJ1A1BXxMaB2Na6taW5a6iuL2y4Y359JvqBGgULRWIc+C9II8egxMWNFzX/5Tz9iporqDxZRG9QLi+zjDDfXjul+QEyzwruuK0qJvH7zgpRnfW46jugcmc5ptrJsJMJoNGI2xSERplDdNFgk0ht8ioii6d4ewQj6+w5z41CmwmjB+DAw3M/Vm+E0MN49wjgrnsfQoRvN8npLu11SZCbHgj4VjLZUK8Ved/jHQEYiN4rz2zNyBSlOhGGk+c2GKkk2yx3nw5ncCBqnGZ5GXrx+hl1bYkq0pQJR0CvL9BQY7p5Q0lItKuQGYi74oZBC4Gm/J50CsheUQWDWNWHfM+3PKO1olleYRnJrt5iNmtXGdklxCtVUiCkx/Waun+okKXtgFegfH/CngelLh11W9O8H4iUQY8Q/eKLyoAthyIT7kcY1HP7yEULBygazqhkPE6ZThD6QuwJa4daaeM4ooUglIScxa+xNooyZUU9IpTjsz2hr8FNASEU5JlIx2EYjvaAcAtZCFhlTJ3S2HD4fuP/5I/gBJRTN7QyP7C9naBWL6tdH+slzlUg3hfqrFYSCkBqZE2SNcpq8yHCZyOOIuXEgC/5LRK4FCkH4OBAfJ4IV2CtHddtQiibnSPzZo5KdYcf7gDSJsb9grjVmoVg9v0VfmRkgfO4hSco5IHVm6Z7BAMUzDxXGExLBeH9GZ8nmZkNSBXUlKETqpMgykKYR7eQvFWQLIbDdviCLTJaR2M0cie78iBQBsVNIK2EqhOOcrC5ZzJWGxuA7Q8oF3TvCpwM4w3A+o64kORekkZRzxl88UmjSaaI/HX8B/U7EPmCExbklzXqBWGgYE+KSUQhU3TCdIv7ek5MiVTA8XciVQMiMP3U0b9boLHHLBt975MZi9YIwjKy2C/R6QRKJRrZIo7CVxZ8C/nFOfdpNjV5K0jSS8IShI/QQTgNiKORjRlkzV1EOI1JKbp8/QzeaJmlsK0FAu1si6oI2GqaI/A8CWaAaKsJlQmvN/ufPpEvBf5prEPk4kbpZ8Z6PA9EUSlcQORCOHtOq2Q4YQQcBWuEfJtJ9YDpfyH2PfbPEOE345T2Qc0Rpg3nZMKZALpKsIqqCcA7o9gq6DoUn7juoM9YJ1ADTvke1hlImTGUQZ81wv+fp3Z48JKgL1WZBkoUkEzkmbp89+9VLTTQCe7VCVRbz/QvSEIh5ThfbxiG1pTcT5TQiREE+s8himD5NuFctxUqmH3rGU6R+vsa+XNG8WaOYQePxv3aYpBFZkTGIa8Xlpz3ySlLfOqrdFXblcK1jejcPeaaHglgIahykSFIzx+b87gvV0uC/9MiuorErQrKYnSCWgXROYAR+3yGvGxpnMGvLZT+yvHlFyYl0EUxeUMZMHzpKdOA0duWgaMLjBTGB2gnkrcZcGtLHkSxGyvvI9OcjeM3lwxPiVrPabKAVyF8Gs8Vmwr5j/497Ql+YeknuRtxtM196TxbTCHKMFK8QgH214PzwNKeCH+fq6OVfHuAZRDKXd0+sfrtCXixNs8KPI7q1tO1cF96+eI66XhEPBSc0BIF9uSINmbTv4TRh
```

Comment:
Can you link to https://xarray-spatial.org/reference/_autosummary/xrspatial.focal.hotspots.html?highlight=hotspots#xrspatial.focal.hotspots?